docs/analyze_index.md

```diff
@@ -3,7 +3,7 @@
 This function calculates the hyperspectral index statistics and writes the values  as observations out to
        the [Outputs class](outputs.md).
        
-**plantcv.hyperspectral.analyze_index**(*index_array, mask*)
+**plantcv.hyperspectral.analyze_index**(*index_array, mask, bins=100, max_bin=None, min_bin=None*)
```

Comment:
```suggestion
**plantcv.hyperspectral.analyze_index**(*index_array, mask, bins=100, min_bin=None, max_bin=None*)
```

I'm only suggesting changing the order because I think people would often list min then max. Doesn't really matter though.